src/main/target/STM32F3DISCOVERY/target.h

```diff
@@ -45,14 +45,14 @@
 
 //#undef USE_TELEMETRY_HOTT
 //#undef USE_TELEMETRY_MAVLINK
-//#undef USE_TELEMETRY_LTM
```

Comment:
Can you just uncomment in place? The list is identical in all targets that use it, and leaving it that way will make future extensions easier.